clash-cores/src/Clash/Cores/Xilinx/Ethernet/Gmii/BlackBoxes.hs

```diff
@@ -0,0 +1,129 @@
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE QuasiQuotes #-}
+module Clash.Cores.Xilinx.Ethernet.Gmii.BlackBoxes where
```

Comment:
```suggestion
{-# LANGUAGE QuasiQuotes #-}

{-# OPTIONS_HADDOCK hide #-}

module Clash.Cores.Xilinx.Ethernet.Gmii.BlackBoxes where
```

Reply:
What is the `OPTIONS_HADDOCK` pragma for?

Reply:
Hiding the completely undocumented module from Haddock. Even if there was documentation it still wouldn't need to be in the Haddock doc.

But I'll also propose a change to the module structure today.